scripts/device-test.ps1

```diff
@@ -21,13 +22,16 @@ $CI = Test-Path env:CI
 Push-Location $PSScriptRoot/..
 try
 {
-    $tfm = 'net8.0-'
+    if (!$Tfm)
+    {
+        $Tfm = 'net8.0'
```

Comment:
once we clear all callsites to pass a tfm, we should delete this and throw an error instead.

Reply:
This is just a default argument, if you don't specify a target framework. We could try to change the default to `latest` or something like that - would need a little bit of extra scripting to work out what latest was.